src/dist/replication/meta_server/meta_service.cpp

```diff
@@ -998,5 +1000,22 @@ void meta_service::on_start_bulk_load(start_bulk_load_rpc rpc)
     }
 }
 
+void meta_service::on_control_bulk_load(control_bulk_load_rpc rpc)
+{
+    if (!check_status(rpc)) {
+        return;
+    }
+
+    if (!_bulk_load_svc) {
+        derror_f("meta doesn't support bulk load");
+        rpc.response().err = ERR_SERVICE_NOT_ACTIVE;
+    } else {
+        tasking::enqueue(LPC_META_STATE_NORMAL,
+                         tracker(),
+                         [this, rpc]() { _bulk_load_svc->on_control_bulk_load(std::move(rpc)); },
```

Comment:
std::move(rpc) has no effect here, because the parameter is not right value in ```bulk_load_service::on_control_bulk_load```.